contrib/inventory/docker_swarm_nodes.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+# (c) 2018, Stefan Heitmueller <stefan.heitmueller@gmx.com>
+#
+# This file is part of Ansible,
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Installation
```

Comment:
Could you please wrap the documentation in a `DOCUMENTATION` block, similar to https://github.com/ansible/ansible/blob/devel/contrib/inventory/docker.py?

Additionally, please split into similar sections: general info, requirements, configuration, and examples. I see that you have most of these, but making it more explicit would be helpful for future users.